src/adaptive.jl

```diff
@@ -0,0 +1,93 @@
+mutable struct Adaptor
+    accepted::Integer
+    total   ::Integer
+    tune    ::Integer   # tuning interval
+    target  ::Float64   # target acceptance rate
+    bound   ::Float64   # bound on logσ of Gaussian kernel
+    δmax    ::Float64   # maximum adaptation step
```

Comment:
The field name and the colons should be flush up against one another with no white space in between, as per our [style guide](https://turing.ml/dev/docs/contributing/style-guide#documentation).

```suggestion
    accepted::Integer
    total::Integer
    tune::Integer     # tuning interval
    target::Float64   # target acceptance rate
    bound::Float64    # bound on logσ of Gaussian kernel
    δmax::Float64     # maximum adaptation step
```